modules/dedicated-creating-your-cluster.adoc

```diff
@@ -25,10 +25,10 @@ following are the default ranges available to use:
 
 .. Pod CIDR: 10.128.0.0/14
 
-. Add your Identity provider by clicking the *Add OAuth Configuration* link.
+. Add your Identity provider by clicking the *Add OAuth Configuration* link or using the *Access Control* tab.
```

Comment:
It's best to only include one method in a step unless there is a clear reason when you'd pick one over the other. I'd remove the first method since you need to be in the *Access Control* tab on the next step.